### ISSUE TITLE
multi-root workspace errors

### ISSUE THREAD

**Issue by offero** [lfurzewaddock/vscode-wsl-workspacefolder#3]
1. Open a folder
2. Add another folder to workspace
3. Run node --inspect-brk
4. Attach to Process using localRoot/remoteRoot with workspaceFolder variables

launch.json:
```
        {
          "type": "node",
          "request": "attach",
          "name": "Attach to Process",
          "address": "localhost",
          "port": 5858,
          "localRoot": "${workspaceFolder}",
          "remoteRoot": "${command:extension.vscode-wsl-workspaceFolder}"
        }
```

I run `node --inspect-brk=5858 ./node_modules/jest/bin/jest` myself in WSL and when I try to attach I get...

`Running the contributed command:'extension.vscode-wsl-workspaceFolder' failed`

**Comment by lfurzewaddock** [comment]
Useful related info: [https://github.com/Microsoft/vscode/wiki/Adopting-Multi-Root-Workspace-APIs](https://github.com/Microsoft/vscode/wiki/Adopting-Multi-Root-Workspace-APIs)

**Comment by lfurzewaddock** [comment]
Fixed by release v1.1.2
